src/usermod.c

```diff
@@ -1342,6 +1366,13 @@ static void process_flags (int argc, char **argv)
 		usage (E_USAGE);
 	}
 
+	if (rflg && (!Gflg)) {
```

Comment:
Should you also enforce that you may not combine -a and -r ?

Reply:
Ahh, my oversight, let me add a check.

Reply:
Added